src/components/WorkerPoolCreateForm.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <p-form class="worker-pool-create-form" @submit="submit">
+    <p-content>
+      <p-label label="Name " :message="nameErrorMessage" :state="nameState">
+        <p-text-input v-model="name" :state="nameState" />
+      </p-label>
+
+      <p-label label="Description (Optional)">
+        <p-textarea v-model="description" rows="7" />
+      </p-label>
+
+      <p-label label="Type" :state="typeState" :message="typeErrorMessage">
+        <p-select model-value="Prefect Agent" :options="['Prefect Agent']" disabled />
+        <!-- Types feature is not implemented by backend yet -->
+        <!-- <WorkerPoolTypeSelect v-model:selected="type" :state="typeState" /> -->
+      </p-label>
+
+      <p-label label="Status (Optional)">
+        <p-toggle v-model="isActive">
+          <template #append>
+            {{ isActiveLabel }}
+          </template>
+        </p-toggle>
+      </p-label>
+
+      <p-label label="Flow Run Concurrency (Optional)">
+        <p-number-input v-model="concurrencyLimit" placeholder="Unlimited" :min="0" />
+      </p-label>
+    </p-content>
+
+    <template #footer>
+      <p-button inset @click="cancel">
+        Cancel
+      </p-button>
+      <SubmitButton action="Create" :loading="pending" />
+    </template>
+  </p-form>
+</template>
+
+<script lang="ts" setup>
+  import { showToast } from '@prefecthq/prefect-design'
+  import { useValidation, useValidationObserver, ValidationRule } from '@prefecthq/vue-compositions'
+  import { computed, ref } from 'vue'
+  import { useRouter } from 'vue-router'
+  import { SubmitButton } from '@/components'
+  import { useWorkspaceApi, useWorkspaceRoutes } from '@/compositions'
+  import { localization } from '@/localization'
+
+  const api = useWorkspaceApi()
+  const router = useRouter()
+  const routes = useWorkspaceRoutes()
+
+  const { validate, pending } = useValidationObserver()
+
+  const name = ref<string>()
+  const description = ref<string>()
+  // Once types feature is implemented by backend, this should be changed to
+  // const type = ref<string>()
+  const type = ref<string>('prefect-agent')
+  const concurrencyLimit = ref<number>()
+  const isActive = ref<boolean>(true)
+  const isActiveLabel = computed(() => isActive.value ? 'Active' : 'Paused')
+
+  const isRequired: ValidationRule<string | undefined> = (value) => value!.trim().length > 0
```

Comment:
if value is undefined, this will error. 
```suggestion
  const isRequired: ValidationRule<string | undefined> = (value) => value !== undefined && value.trim().length > 0
```